test/unix/dummy.ml

```diff
@@ -1,6 +1,26 @@
+let str = "the quick brown fox jumps over the lazy dog"
+let bytes = Bytes.of_string str
+let len = Bytes.length bytes
+
+let read () =
+  let buf = Bytes.create len in
+  let rec aux n =
+    let i = Unix.read Unix.stdin buf n (Bytes.length buf - n) in
+    if n + i = len then
+      Bytes.compare buf bytes = 0
+    else aux (n + i)
+  in
+  if aux 0 then begin
+    (* try to read a bit more, should raise broken pipe *)
+    ignore (Unix.read Unix.stdin buf 0 1);
```

Comment:
Why do this?

Reply:
It's not really needed, but I just want to make sure the test is reading only the `test_input_str` and nothing more.